build.sbt

```diff
@@ -1,11 +1,11 @@
 name := "consul-rest-client"
 organization := "com.github.dcshock"
-version := "1.1"
+version := "1.2"
```

Comment:
@toddbyrne - I think this is the other spot, no?

Reply:
Yeah that is the spot.